tests/test_errors.py

```diff
@@ -119,7 +119,11 @@ def validate(self, input_value, info):
 
     c = CustomValidator()
     v = SchemaValidator(
-        {'type': 'function-plain', 'metadata': {'instance': c}, 'function': {'type': 'general', 'function': c.validate}}
+        {
+            'type': 'function-plain',
+            'metadata': {'instance': c},
+            'function': {'type': 'with-info', 'function': c.validate},
+        }
```

Comment:
Please use the `core_schema` function instead of a plain dict

Reply:
done.